google-cloud-datastore/lib/google/cloud/datastore/service.rb

```diff
@@ -55,7 +55,7 @@ def service
           return mocked_service if mocked_service
           @service ||= V1::DatastoreClient.new(
             service_path: host,
-            channel: channel,
+            credentials: channel,
```

Comment:
Just making sure this is correct?

Reply:
Yes, this is correct. For Datastore and Pub/Sub we need to pass in a GRPC::Channel because the user might be using an emulator instead of connecting to the Datastore API. But we need to use the `credentials` argument name to avoid a warning from the most recent GAPIC changes.